testutil/src/main/java/org/spine3/test/Verify.java

```diff
@@ -1508,7 +1508,9 @@ public static void assertThrows(
      * <p>
      * e.g.
      * <pre>
-     * Verify.assertThrowsWithCause(RuntimeException.class, IOException.class, new Callable<Void>()
+     * Verify.assertThrowsWithCause(RuntimeException.class,
+     *                              IOException.class,
+     *                              new Callable&lt;Void&gt;()
```

Comment:
Please fix the code formatting in this snippet regarding the Spine code style conventions.